stable/listmonk/templates/init-db.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: batch/v1
```

Comment:
Thanks, yes wanted to check if it works or not, after moving job to pre-install hook it will not be able to do this, shall I disable chart deployment CI for this chart?